pyproject.toml

```diff
@@ -0,0 +1,37 @@
+[tool.poetry]
+name = "multi-start"
+version = "0.0.2"
+description = "Run multiple services inside a docker container"
+license = "BSD-3-Clause"
+readme = "README.md"
+repository = "https://github.com/ioxiocom/multi-start"
+authors = ["IOXIO Ltd"]
+packages = [
+    {include="src", from="."}
+]
+
+[tool.poetry.dependencies]
+python = "^3.10"
+typer = "^0.9.0"
+httpx = "^0.24.1"
+pydantic = "^1.10.9"
+
+[tool.poetry.dev-dependencies]
```

Comment:
I believe this is an old legacy thing that you could remove, since you've already switched to the `[tool.poetry.group.dev.dependencies]`.